rust/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "tsnet"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[features]
+default = ["tokio"]
```

Comment:
TODO: make tokio optional but off by default when I am done writing and testing the async code. This is enabled by default for development convenience.